src/interface/common.thrift

```diff
@@ -103,9 +103,20 @@ struct Pair {
 }
 
 struct IndexItem {
-     1: required IndexID          index_id,
-     2: required i32              tagOrEdge,
-     3: required list<ColumnDef>  cols,
+    1: required IndexID          index_id,
+    2: required i32              tagOrEdge,
+    3: required list<ColumnDef>  cols,
+}
+
+struct IndexHintItem {
+    1: binary                first_str,
```

Comment:
what is the first and second string’s meaning ？

Reply:
> what is the first and second string’s meaning ？

 Aha , I'm refactoring the index scan logic now. Most of the existing code will be discarded. 
By the way, the first and second string is a data scan range-pair,  I'll use the native expression instead of it in the storage layer.

Reply:
> what is the first and second string’s meaning ？


what is the first and second string’s meaning ？

The index scan filter logic push down to storage layer.